press/views/api-docs/swagger.yaml

```diff
@@ -43,6 +43,71 @@ tags:
       because they are meant only for use within the browser
       rather than as actual API usage.
 paths:
+  '/ping':
+    get:
+      summary: Verifies that press server is running
+      description: ''
+      operationId: ping
+      produces:
+        - text/plain
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            type: string
+      tags:
+        - system
+  '/api/ping':
+    get:
+      summary: Verifies that press server is running
+      description: ''
+      operationId: apiping
+      produces:
+        - text/plain
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            type: string
+      tags:
+        - system
+  '/api/auth-ping':
+    get:
+      summary: Verifies that authenticaton succeeds
+      description: ''
+      operationId: authping
+      produces:
+        - text/plain
+        - application/json
```

Comment:
Curious why is this the only `path` that produces JSON in addition to text/plain.

Reply:
Yep, but wouldn't the others behave the same way so they should also be able to produce JSON?

Reply:
yup, I updated that for the `/api*ping` paths that can potentially error (added `assert`s as well on the content_type in the response)